packages/slate-plugins/src/elements/mention/transforms/wrapMentionBrackets.ts

```diff
@@ -0,0 +1,63 @@
+import { Point, Range, Editor, Transforms } from "slate";
+
+const pointToStr = (point: Point | undefined) => {
```

Comment:
`point? :Point` should work here

Reply:
is it important to make this change? seems fine as is. lmk if referring to some convention.

Reply:
I think it's best to follow existing convention in tht codebase, whatever it is.